lib/acts-as-taggable-array-on/taggable.rb

```diff
@@ -18,6 +28,16 @@ def acts_as_taggable_array_on(tag_name, *)
         scope :"without_any_#{tag_name}", ->(tags) { where.not("#{table_name}.#{tag_name} && ARRAY[?]::#{tag_array_type_fetcher.call}[]", parser.parse(tags)) }
         scope :"without_all_#{tag_name}", ->(tags) { where.not("#{table_name}.#{tag_name} @> ARRAY[?]::#{tag_array_type_fetcher.call}[]", parser.parse(tags)) }
 
+        if args[:allow_list].present?
```

Comment:
Can we extract `:allow_list` into a constant?

It seems reasonable to do, since we repeat this at least 3 times in method definition. In case we need to switch this value, we'll have lower chance of typo mistake.

Reply:
If I understand correctly what you suggested, I did in this commit https://github.com/tmiyamon/acts-as-taggable-array-on/pull/40/commits/8c1aa9d4eef9e5c245900a51ab68598c6845feca